src/main/java/com/project/exception/AdminCommentNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package com.project.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
+public class AdminCommentNotFoundException extends Exception {
+    public AdminCommentNotFoundException(long id) {
+        super("Komentaras su nurodytu identifikatoriumi (" + id + ") neegzistuoja.");
```

Comment:
You could use `super(String.format("Komentaras su nurodytu identifikatoriumi (%d) neegzistuoja.", id))`. Looks a bit cleaner.

Reply:
changed validation messages to use `String.format()`